src/background/data/plannerdata.js

```diff
@@ -506,13 +545,14 @@ window.PlannerData = {
 
     Revenant:{
         core: [
+            //Awakening
             new PlannerItem(1, SUPPLYTYPE.treasure, PLANNER_TEMPLATES.orbs.high.Light, 50),
             new PlannerItem(1, SUPPLYTYPE.treasure, PLANNER_TEMPLATES.scrolls.Light, 50),
             new PlannerItem(1, SUPPLYTYPE.treasure, PLANNER_TEMPLATES.whorls.Light, 50),
             new PlannerItem(1, SUPPLYTYPE.treasure, 1151, 50),
             new PlannerItem(1, SUPPLYTYPE.treasure, 2001, 50),
-//            createItem(1, SUPPLYTYPE.treasure, 1052, 50),
-
+            //crystals 100
```

Comment:
Crystal tracking was pushed a few commits ago, would be great if you could add those so Revenants as a whole is up to date with where Veera is currently.

Reply:
There's a SUPPLYTYPE for them, and their id is 0 iirc.